taskfiles/rp.yml

```diff
@@ -128,6 +128,9 @@ tasks:
     cmds:
     - |
       PATH={{.BUILD_ROOT}}/bin/:$PATH
+      if ! {{empty .USE_PODMAN_DOCKER}}; then
+        export DOCKER_HOST=unix://$XDG_RUNTIME_DIR/podman/podman.sock
```

Comment:
xdg runtime dir is defined already?

Reply:
yeah, that's part of the freedesktop spec. All mainstream distros define these XDG_* variables.

Reply:
amazing. 

feel free tomerge